bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -14,18 +14,16 @@ class Steps(CMakePackage):
     homepage = "https://groups.oist.jp/cnu/software"
     git = "ssh://git@bbpgitlab.epfl.ch/hpc/HBP_STEPS.git"
 
-    submodules = True
+    maintainers("tristan0x")
 
-    version("develop", branch="master")
-    version("5.0.0a", commit="d75d46f")
-    version("4.0.0", tag="4.0.0", preferred=True)
-    version("3.6.0", tag="3.6.0")
-    version("3.5.0b", commit="b2be5fe")
+    version("develop", branch="master", submodules=True)
+    version("5.0.0a", commit="b4bdd64", submodules=True)
+    version("4.1.0", submodules=True, preferred=True)
```

Comment:
```suggestion
    submodules = True

    version("develop", branch="master")
    version("5.0.0a", commit="b4bdd64")
    version("4.1.0", tag="4.1.0", preferred=True)
```
Also why does this not have a tag?

Reply:
> Also why does this not have a tag?

Why isn't it the default? 😆 
/jk, will report this upstream. There should be a nice error message for retards like me.